### PR TITLE
feat: case-insensitive shc protocol prefix removal

### PR DESCRIPTION
**Pull request by albertxing**
iOS counterpart: https://github.com/bcgov/BCVAX-iOS/pull/10

> Some QR code generators / display apps can only encode the SHC token efficiently if shc:/ is in upper case, this ensures the validator app can read these encoded QR codes.

Note: Because only the `shc:/` prefix is alphabetical, and the rest of the URI consists of digits, converting the entire string to lowercase should be safe

### CODE REVIEW

**Review by will-freshworks — APPROVED** [review]

**Comment by albertxing** [comment]
I don't have merge permissions, are you able to merge this @will-freshworks?

**Comment by will-freshworks** [comment]
> I don't have merge permissions, are you able to merge this @will-freshworks?

@amit-metri could you merge this for the next release if you're happy with it?

**Comment by inthewaves** [comment]
FYI, one of my earlier MRs was closed because it targeted `main` instead of `develop`: https://github.com/bcgov/BCVAX-Android/pull/25

**Pull request closed without merging**
